finality-aleph/src/party/mod.rs

```diff
@@ -157,20 +124,20 @@ where
     let session_authorities = Arc::new(Mutex::new(HashMap::new()));
     let block_requester = network.clone();
 
+    let justification_handler_config: JustificationHandlerConfig<_> = Default::default();
```

Comment:
The typing here shouldn't be required anymore.

Reply:
Done